src/platform/forms/save-in-progress/ApplicationStatus.jsx

```diff
@@ -150,7 +155,7 @@ export class ApplicationStatus extends React.Component {
             </p>
             {multipleForms && (
               <p className="no-bottom-margin">
-                You have more than one in-progress {formType} form.{' '}
+                You have more than one in-progress {formType} {appType}.{' '}
```

Comment:
@rianfowler This is the first I'm seeing of this `formType`. I wasn't able to debug it in the app; do you know what this is supposed to be?